content/ja/handbook/engineering/infrastructure-platforms/tenant-scale/gitaly/index.md

```
@@ -0,0 +1,314 @@
+---
+title: "Gitaly チーム"
+upstream_path: "/handbook/engineering/infrastructure-platforms/tenant-scale/gitaly/"
+upstream_sha: "0e6f01390a34aeb6706ace17d8d3c50e74e82d0d"
+translated_at: "2026-04-28T00:00:00Z"
+translator: claude
+stale: false
+---
+
+## Gitaly とは何か？
+
+Gitaly チームは、GitLab インスタンスの Git データストレージ層、_特に GitLab.com_ が、信頼性が高く、安全で高速であることを確保するためのシステムを構築・保守する責任があります。Gitaly の詳細については、[リポジトリの README](https://gitlab.com/gitlab-org/gitaly/-/blob/master/README.md) と以下の[ロードマップ](#roadmap)を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)